setup/quickstart/halyard-gce.md

```diff
@@ -0,0 +1,240 @@
+---
+
+layout: single
+title:  "Halyard on GCE Quickstart"
+sidebar:
+  nav: setup
+---
+
+{% include toc %}
+
+In this guide we'll be going through the basics of deploying Spinnaker to a
+single VM instances on GCE; however, many of the operations here apply to any
+VM running Ubuntu 14.04.
+
+## Overview
+
+Once we are finished we will have the following setup:
+
+* Spinnaker running on a GCE VM, installed using each subservices Debian
+  package.
+* Persistent storage configured to use GCS.
+* Spinnaker configured to deploy resources to GCE, and bake GCE VM images.
+
+## Part 0: Preparation
+
+### Install gcloud
+
+If you don't already have gcloud installed, navigate to [Installing Cloud SDK](https://cloud.google.com/sdk/downloads#interactive) to install gcloud
+
+### Enable the GCP IAM API
+
+Navigate to the [Google Cloud Console](https://console.developers.google.com/apis/api/iam.googleapis.com/overview) and enable the Google Identity and Access Management (IAM) API
+
+### Set up credentials
+
+Create a service account for our Halyard host VM:
+
+```bash
+GCP_PROJECT=$(gcloud info --format='value(config.project)')
+HALYARD_SERVICE_ACCOUNT_NAME=halyard-vm-account
+
+gcloud iam service-accounts create $HALYARD_SERVICE_ACCOUNT_NAME \
+    --project=$GCP_PROJECT \
+    --display-name $HALYARD_SERVICE_ACCOUNT_NAME
+
+HALYARD_SERVICE_ACCOUNT_EMAIL=$(gcloud iam service-accounts list \
+    --project=$GCP_PROJECT \
+    --filter="displayName:$HALYARD_SERVICE_ACCOUNT_NAME" \
+    --format='value(email)')
+
+# permission to download service account keys
+gcloud projects add-iam-policy-binding $GCP_PROJECT \
+    --role roles/iam.serviceAccountKeyAdmin \
+    --member serviceAccount:$HALYARD_SERVICE_ACCOUNT_EMAIL
+```
+
+Create a service account for GCS that you'll later be handing to Spinnaker
+
+```bash
+GCP_PROJECT=$(gcloud info --format='value(config.project)')
+GCS_SERVICE_ACCOUNT_NAME=spinnaker-gcs-account
+
+gcloud iam service-accounts create $GCS_SERVICE_ACCOUNT_NAME \
+    --project=$GCP_PROJECT \
+    --display-name $GCS_SERVICE_ACCOUNT_NAME
+
+GCS_SERVICE_ACCOUNT_EMAIL=$(gcloud iam service-accounts list \
+    --project=$GCP_PROJECT \
+    --filter="displayName:$GCS_SERVICE_ACCOUNT_NAME" \
+    --format='value(email)')
+
+# permission to create/modify buckets in your project
+gcloud projects add-iam-policy-binding $GCP_PROJECT \
+    --role roles/storage.admin \
+    --member serviceAccount:$GCS_SERVICE_ACCOUNT_EMAIL
+```
+
+```bash
```

Comment:
I'd add some text describing what this script is doing, just like what you did above with storage